pkgs/os-specific/darwin/apple-source-releases/Libsystem/default.nix

```diff
@@ -31,9 +31,11 @@ appleDerivation {
 
     mkdir -p $out/include/os
 
-    cp ${darling.src}/src/libc/os/activity.h $out/include/os
-    cp ${darling.src}/src/libc/os/log.h $out/include/os
-    cp ${darling.src}/src/duct/include/os/trace.h $out/include/os
+    (cd ${darling.src}/platform-include && (xargs -n1 echo | cpio -pdm --quiet $out/include) <<<"
+      os/activity.h
+      os/log.h
+      os/trace.h
+    ")
```

Comment:
/nix/store/pawa4j5ywf0s0mm1khk8f45qx8qrj49v-Libsystem-1238.60.2-osx-10.12.6/include/os/trace.h:32:10: fatal error: 'os/trace_base.h' file not found
#include <os/trace_base.h>
         ^~~~~~~~~~~~~~~~~

Reply:
I think this header was added in SDK 10.13. Compare:
https://github.com/linxun45/MacOSX-SDKs/tree/aea47c83334af9c27dc57c49ca268723ef5e6349/MacOSX10.12.sdk/usr/include/os
vs
https://github.com/linxun45/MacOSX-SDKs/tree/aea47c83334af9c27dc57c49ca268723ef5e6349/MacOSX10.13.sdk/usr/include/os

Reply:
Here is a little complicated, `os/trace_base.h` required `os/availability.h` (not the one below, a new one), while the new `os/availability.h` required new `Availability.h` in `xnu` according to https://github.com/NixOS/nixpkgs/pull/110685#pullrequestreview-576682236 . We have to do it step by step to keep minimal change in each PR.

@veprbl would you reviewed #109368 first? Then I will try making a pure xnu update, then darling.